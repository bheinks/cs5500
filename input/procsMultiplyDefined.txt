(*
	Procedure names multiply defined (same names in 
      same scope)
*)

program p;

procedure proc;
begin
  write(99)
end;

procedure proc;
begin
  write(100)
end;

begin
  write(5)
end.

