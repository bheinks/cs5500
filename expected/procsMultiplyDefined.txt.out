

>>> Entering new scope...

+++ Adding p to symbol table with type PROGRAM

+++ Adding proc to symbol table with type PROCEDURE


>>> Entering new scope...

<<< Exiting scope...

+++ Adding proc to symbol table with type PROCEDURE
Line 13: Multiply defined identifier
